src/controllers/workspace/getWorkspaces.ts

```diff
@@ -0,0 +1,15 @@
+import Workspace from '../../models/Workspace'
+
+async function getWorkspaces(req: any, res: any) {
+  try {
+    const workspaces = await Workspace.find({})
+    if (workspaces.length === 0) {
```

Comment:
By default, the workspace array is going to be empty until the user creates a workspace, so the length is going to be 0. If we say that the server should return a 404 when that happens, that might cause problems for us on the frontend. 

A more suitable approach would be if(workspaces.length === undefined || workspaces === null) then throw a 404